frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/table/EnhancedGraphTable.java

```diff
@@ -57,14 +56,23 @@
 import com.virtuslab.gitmachete.frontend.graph.api.repository.IRepositoryGraphCache;
 import com.virtuslab.gitmachete.frontend.graph.api.repository.NullRepositoryGraph;
 import com.virtuslab.gitmachete.frontend.ui.api.gitrepositoryselection.IGitRepositorySelectionProvider;
-import com.virtuslab.gitmachete.frontend.ui.api.table.BaseGraphTable;
+import com.virtuslab.gitmachete.frontend.ui.api.table.AbstractEnhancedGraphTable;
 import com.virtuslab.gitmachete.frontend.ui.impl.cell.BranchOrCommitCell;
 import com.virtuslab.gitmachete.frontend.ui.impl.cell.BranchOrCommitCellRendererComponent;
 import com.virtuslab.gitmachete.frontend.ui.providerservice.SelectedGitRepositoryProvider;
 
+/**
+ *  {@link EnhancedGraphTable} compared to {@link SimpleGraphTable} has graph table refreshing and provides
```

Comment:
Hmm this comment should rather go to `AbstractEnhancedGraphTable` instead

Reply:
At least the part about `graph table refreshing`... the other things are `EnhancedGraphTable`-specific indeed :thinking:

Reply:
Ok, so I'v add proper javadoc to `BaseEnhancedGraphTable` also (previously named `AbstractEnhancedGraphTable`)